Firmware/mmu2.cpp

```diff
@@ -638,27 +639,22 @@ StepStatus MMU2::LogicStep() {
         break;
     case CommandError:
         ReportError(logic.Error());
-        CheckUserInput();
         break;
     case CommunicationTimeout:
         state = xState::Connecting;
         ReportError(ErrorCode::MMU_NOT_RESPONDING);
-        CheckUserInput();
         break;
     case ProtocolError:
         state = xState::Connecting;
         ReportError(ErrorCode::PROTOCOL_ERROR);
-        CheckUserInput();
         break;
     case VersionMismatch:
         StopKeepPowered();
         ReportError(ErrorCode::VERSION_MISMATCH);
-        CheckUserInput();
         break;
     case ButtonPushed:
         lastButton = logic.Button();
         LogEchoEvent("MMU Button pushed");
-        CheckUserInput();
```

Comment:
This one might need to be left alone otherwise the MMU button won't be registered until the next cycle, if at all.

Reply:
I've added this back